### DISCUSSION TITLE
How to extract zip file be able to view the contents delete after

### DISCUSSION THREAD

**Discussion by skeept** [gokcehan/lf#1663] [Q&A] [answered]
I am wondering if it would be possible to do the following: 

if the file under the cursor is an archive (let's say zip) then I would like for the file to be extracted to a temporary location when I click the right key and go into that folder with the extracted files.
Once I am done viewing that folder and go back then folder would be deleted. Another possibility would be to delete all the extracted folder and exit time.

Would this be possible?

**Comment by DusanLesan** [comment, marked as answer]
That is very similar to [archivemount integration](https://github.com/gokcehan/lf/wiki/Integrations#archivemount). While it maps the command to new shortcut, you can easily add it to open command like I did in [my config](https://github.com/DusanLesan/dotfiles/blob/289c8dc1c4a4639ff00b518ef5c34994b4a349f3/.config/lf/lfrc#L15)
